src/test/java/io/strimzi/kafka/bridge/http/SeekTest.java

```diff
@@ -64,15 +50,24 @@ void seekToBeginningConsumerOrPartitionNotFound(VertxTestContext context) throws
                         assertEquals(HttpResponseStatus.NOT_FOUND.code(), response.statusCode());
                         assertEquals(HttpResponseStatus.NOT_FOUND.code(), error.getCode());
                         assertEquals("The specified consumer instance was not found.", error.getMessage());
+                        context.completeNow();
                     });
-                    instanceNotFound.complete(true);
                 });
-        instanceNotFound.get(TEST_TIMEOUT, TimeUnit.SECONDS);
+        assertTrue(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS));
+    }
+
+    @Test
+    @Disabled // This test was disabled because of known issue described in https://github.com/strimzi/strimzi-kafka-bridge/issues/320
+    void seekToNotExistentPartitionInSubscribedTopic(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
+        String topic = "seekToNotExistentPartitionInSubscribedTopic";
+        kafkaCluster.createTopic(topic, 1, 1);
 
-        // Specified consumer instance did not have one of the specified partitions assigned.
-        CompletableFuture<Boolean> consumerInstanceDontHavePartition = new CompletableFuture<>();
+        // create consumer
+        consumerService()
+            .createConsumer(context, groupId, jsonConsumer)
+            .subscribeTopic(context, groupId, name, new JsonObject().put("topic", topic).put("partition", 0));
 
-        int nonExistenPartition = Integer.MAX_VALUE;
+        int nonExistenPartition = 2;
```

Comment:
```suggestion
        int notExistingPartition = 2;
```

Reply:
Kafka Topic has only 1 partition... so it's more clear to me to have value 2 here

Reply:
why did you think that I wanted to change the value? :-) I changed just the name. It was you changing ffrom MAX_VALUE to 2 :P